src/main/java/frc/robot/subsystems/vision/VisionIO.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.subsystems.vision;
+
+import java.util.Optional;
+
+import org.photonvision.EstimatedRobotPose;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+
+import edu.wpi.first.math.geometry.Pose2d;
+
+public interface VisionIO {
```

Comment:
Can we also make this AutoLogged (or whatever makes sense in AdvantageKit)?

Reply:
fixed to make auto log by moving majority of methods to update inputs method. Also added array of target corners (x/y values). Not sure when we would use those but advantage kit had it used in their vision example so i added just in case.

Reply:
looking further over their example, it looks like they used to establish camera to target translation for pose estimation. Photon vision has that built in now so it should not be needed. Should i remove?

Reply:
I think we only need the camera pose estimate itself - it's best if we can outsource the work to the co-processor running photonvision in this case